lib/utils.js

```diff
@@ -141,11 +141,34 @@ function updateSourceLocation(spec) {
   }
 }
 
+/**
+   * Determine if a given filePath is a license file based on name
+   * Checks deeper than the root depending on coordinate type
+   *
+   * @param {string} filePath
+   * @param {EntityCoordinates} coordinates - optional to look deeper than the root based on coordinate type
+   * @returns {boolean}
+   */
+function isLicenseFile(filePath, coordinates) {
+  if (!filePath) return false
+  filePath = filePath.toLowerCase()
+  const basePath = filePath.split('/')[0]
+  if (_licenseFileNames.includes(basePath)) return true
+  if (!coordinates) return false
+  for (const prefix of _licenseLocations[coordinates.type] || []) {
+    if (_licenseFileNames.includes(filePath.replace(prefix, ''))) return true
+  }
+  return false
+}
+
 function _normalizeVersion(version) {
   if (version == '1') return '1.0.0' // version '1' is not semver valid see https://github.com/clearlydefined/crawler/issues/124
   return semver.valid(version) ? version : null
 }
 
+const _licenseFileNames = ['license', 'license.txt', 'license.md', 'license.html']
+const _licenseLocations = { 'npm': ['package/'] }
```

Comment:
know of any other prefixes we should add here off hand?

Reply:
'maven': 'meta-inf/'

Note as well that we should likely expand the set of "license" files. Some communities use "COPYING" for example. @pombredanne  would be a great source of a list of typical filenames containing license declarations for different package types.